index.js

```diff
@@ -148,18 +148,25 @@ async function cloneExample() {
     await run(`git clone --depth 1 --filter=blob:none --sparse ${config.examples.repoUrl} ${tmpDir}`);
     // Checkout just the example dir.
     await run(`cd ${tmpDir} && git sparse-checkout set ${args.example}`);
-    // Copy out into a new directory within current working directory.
-    await run(`cp -R ${tmpDir}/${args.example} ${dirName}`);
+
```

Comment:
This is fine, but generally, when you're changing code, it's not necessary to adjust formatting, including adding new lines, unless there's a specific reason for doing it.

Reply:
This is my formatter's mistake, I enforced very specific rules. I'll update